File: app-service-comments/src/main/java/com/github/arhor/aws/graphql/federation/comments/util/Caches.java
package com.github.arhor.aws.graphql.federation.comments.util;

public enum Caches {
    /**
     * Cache for preparsed GraphQL documents.
     */
    GRAPHQL_DOCUMENTS,

    /**
     * Cache for idempotent operation execution.
     */
    IDEMPOTENT_ID_SET,
}

File: app-service-posts/src/main/kotlin/com/github/arhor/aws/graphql/federation/posts/util/Caches.kt
package com.github.arhor.aws.graphql.federation.posts.util

enum class Caches {
    /**
     * Cache for preparsed GraphQL documents.
     */
    GRAPHQL_DOCUMENTS,

    /**
     * Cache for idempotent operation execution.
     */
    IDEMPOTENT_ID_SET,
}
